LayoutTests/imported/w3c/web-platform-tests/scroll-animations/css/pseudo-on-scroller-named-timeline.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Animating pseduo-element on scroller using named timeline</title>
+</head>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/web-animations/testcommon.js"></script>
+<script src="./support/testcommon.js"></script>
+<script src="testcommon.js"></script>
+<style type="text/css">
+.scroller {
+  overflow: scroll;
+  width: 100px;
+  height: 100px;
+  margin: 1em;
+  outline: 1px solid;
+  scroll-timeline: --t1 y;
+}
+.pseudo::before {
+  content: "";
+  display: block;
+  width: 50px;
+  height: 50px;
+  background: red;
+  animation: bg linear reverse;
```

Comment:
Is `reverse` here required for this test?

Reply:
Not strictly no.